src/ad/azext_ad/azext_metadata.json

```diff
@@ -0,0 +1,3 @@
+{
+    "azext.minCliCoreVersion": "2.15.0"
+}
```

Comment:
This is against the coding convention that any source code file should end with an empty line. @qiaozha

Reply:
This is against the coding convention that any source code file should end with an empty line.